tests/integration/cache/test_utils.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Test lock cache."""
+
+from __future__ import absolute_import, division, print_function
+
+
+from inspirehep.modules.cache.utils import redis_locking_context
+
+
+def test_redis_locking_context_write_string_in_redis(app):
+    with redis_locking_context('my_lock') as redis:
+        redis.set('key', 'value')
+        assert redis.get('key') == 'value'
+
+
+def test_redis_locking_context_write_integer(app):
+    with redis_locking_context('my_lock') as redis:
+        redis.set('key', 1)
+        assert redis.get('key') == '1'
```

Comment:
I'm not sure these tests are useful here, they are testing if you can put stuff and retrieve stuff from redis.

I would either create unit tests, and assert that the lock is called (not very nice, though might be enough), or integration tests that actually check if the lock is put in redis.